addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -122,8 +137,8 @@ var SmsWidget = FieldTextEmojis.extend({
      * @private
      */
     _renderSMSInfo: function () {
-        var string = _.str.sprintf(_t('%s characters, fits in %s SMS (%s) '), this.nbrChar, this.nbrSMS, this.encoding);
-        var $span = $('<span>', {
+        const string = _.str.sprintf(this._getCountString(), this.nbrChar, this.nbrSMS, this.encoding);
```

Comment:
Isn't it better to extract the complete string formation into a submethod, instead of just the string to format itself ? That way you can reorder arguments for example.